src/pages/EnablePayments/OnfidoStep.tsx

```diff
@@ -64,7 +63,7 @@ function OnfidoStep({walletOnfidoData = DEFAULT_WALLET_ONFIDO_DATA}: OnfidoStepP
                 onBackButtonPress={goToPreviousStep}
             />
             <FullPageOfflineBlockingView>
-                {shouldShowOnfido ? (
+                {shouldShowOnfido && walletOnfidoData?.sdkToken ? (
                     <Onfido
                         sdkToken={walletOnfidoData.sdkToken}
```

Comment:
`walletOnfidoData.sdkToken` is already included in `shouldShowOnfido` check.

To make TS happy:
```suggestion
                {shouldShowOnfido ? (
                    <Onfido
                        sdkToken={walletOnfidoData.sdkToken ?? ''}
```

Reply:
Thanks for the review and suggestion @situchan  ,added changes.